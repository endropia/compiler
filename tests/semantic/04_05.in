 var
 	a: string;
 begin
 	writeln(not a);
 end.