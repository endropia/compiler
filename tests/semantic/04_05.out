Unary operator does not overloaded